pkg/gitproviders/provider_test.go

```diff
@@ -424,6 +457,85 @@ var _ = Describe("test personal repo exists", func() {
 
 })
 
+var _ = Describe("test gitlab project", func() {
```

Comment:
Because this has the same issue with go-git-providers that the acceptance test had. This issue is written up here https://app.zenhub.com/workspaces/weaveworks-foss-603cb6b1d4dff9000e242dc4/issues/weaveworks/weave-gitops/808

Reply:
Copy that, ➕ for including the link to the issue.  Thanks!!!